<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeCode</class>
 <widget class="QWidget" name="ChangeCode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_text_originalcode">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="index.qrc">:/Changecode/src/ORIGINALCODE_W_100_50.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_text_newcode">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="index.qrc">:/Changecode/src/NEWCODE_W_100_50.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_text_confirmcode">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="index.qrc">:/Changecode/src/CONFIRMCODE_W_100_50.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_return">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>520</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image:url(:/Return/src/RETURN_W.png)}
QPushButton:hover{border-image:url(:/Return/src/RETURN_P.png) }
QPushButton:pressed{border-image: url(:/Return/src/RETURN_C.png)}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_changecode">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>410</y>
     <width>150</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image:url(:/Changecode/src/CONFIRMCHANGE_W.png)}
QPushButton:hover{border-image:url(:/Changecode/src/CONFIRMCHANGE_P.png) }
QPushButton:pressed{border-image:url(:/Changecode/src/CONFIRMCHANGE_C.png) }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_originalcode">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>212</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_newcode">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>282</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_confirmcode">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>352</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="index.qrc"/>
 </resources>
 <connections/>
</ui>
